specification/streamanalytics/resource-manager/readme.md

```diff
@@ -4,33 +4,80 @@
 
 This is the AutoRest configuration file for StreamAnalytics.
 
-
-
 ---
+
 ## Getting Started
+
 To build the SDK for StreamAnalytics, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the StreamAnalytics API.
 
 ``` yaml
 title: Stream Analytics Management Client
 description: Stream Analytics Client
 openapi-type: arm
-tag: package-2016-03
+tag: package-2020-03-preview
 ```
 
+### Tag: package-2020-03-preview
+
+These settings apply only when `--tag=package-2020-03-preview` is specified on the command line.
+
+This section contains the composite set of APIs. Deticate cluster and private endpoint related APIs are under 2020-03-01-preview version. Streamingjobs related APIs are under 2017-04-01-preview version.
+
+```yaml $(tag) == 'package-2020-03-preview'
+input-file:
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/functions.json
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/inputs.json
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/outputs.json
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/streamingjobs.json
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/subscriptions.json
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/transformations.json
+  - Microsoft.StreamAnalytics/preview/2017-04-01-preview/operations.json
+  - Microsoft.StreamAnalytics/preview/2020-03-01-preview/clusters.json
+  - Microsoft.StreamAnalytics/preview/2020-03-01-preview/privateEndpoints.json
+```
+
+### Tag: package-pure-2020-03-preview
+
+These settings apply only when `--tag=package-pure-2020-03-preview` is specified on the command line.
+
+This section only contains input swagger files for version 2020-03-01-preview, incluing deticate cluster and private endpoint related APIs.  
```

Comment:
including

I see this typo in multiple places as well

Reply:
Fixed